docs/redesign-specs/DB_Schema/database.md

```diff
@@ -0,0 +1,199 @@
+**ASSUMPTION:** A reliable underlying technology that provides 100% availablity of the database, following are some examples of the relations between different tables. 
+
+
+All the tables in the BMI database are listed bellow: 
+
+## User 
+
+1. An entity using BMI. 
+2. Users will be registered to BMI by an admin via the API after BMI has been deployed. 
+
+* id : Id of the user (primary key)
+* name : Name of the user (unique key) 
+* type : Type of user (ordinary user or admin)
+* quota : Quantity of datastore disk-space allocated to a user (GB)
+* status : Current status of user (e.g. active, inactive, disabled, etc.)

```

Comment:
What does this mean?  How can you tell if someone is active?  If you have any machines currently backed by BMI then it is active, so user stays active for days/weeks/months?  What is purpose of this?